### PR TITLE
Clean up about page and move compatibility matrix

### LINKED ISSUE #1128: Focus /about page a little
The current ```/about``` page is a bit of a hotch-potch of information about the site's purpose, use, contributors, compatibility and contact information. It would be great if this information was more clearly presented or - better yet - split up. 

In general I think this page is serving the who, what, where and why when  I think it only need serve the what. 

My proposal would be to keep ```/about``` as a page containing 
* what the site is
* what you can use it for
* (optionally) the compatibility matrix

with links out to:

* ```/who``` which provides information on supporters and contributors and a 'funnel' to guide people toward contributing. 
* ```/why``` which contains a precis of our strategy and rationale for building libraries.io
* ```/contact``` which contains a contact form and some support information.

### PR DESCRIPTION
**Pull request by BenJam**
* break out _who and _contact partials
* re-write /about page to be more usage focussed
* add section on strategy 
* add section on contributing 

Closes #1128

### CODE REVIEW

File: app/views/pages/pmmatrix.html.erb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    <p>
+      Some package managers require compilation before parsing their dependencies, some we simply haven't got around to yet. Luckily Libraries.io is an open source project so if you need to add a new package manager then you can. You can find everything you need to get started in <a href='https://docs.libraries.io' target='_blank'>our documentation</a>.
+    </p>
+  </div>
+  <div class="col-sm-4">
+    <%= render 'adsense/sidebar' %>
+    <%= render 'contact' %>
+  </div>
+</div>
+<!-- <div class="row">
```
Comment (andrew) [inline app/views/pages/pmmatrix.html.erb]: No need to keep this here

File: app/views/pages/about.html.erb [hunk 2]
```diff
@@ -65,56 +78,51 @@
...
-    <% end %><br>
-    <%= link_to 'https://twitter.com/benjam' do %>
-      <%= fa_icon 'twitter' %> Twitter
-    <% end %>
+    <%= render 'contact' %>
+    <%= render 'who' %>
   </div>
 </div>
-<div class="row">
+<!-- <div class="row">
```
Comment (andrew) [inline app/views/pages/about.html.erb]: No need to keep this here

File: app/views/pages/about.html.erb [hunk 3]
```diff
@@ -1,56 +1,69 @@
...
-      <strong>Advanced search</strong>
+      <strong>Monitor Your Projects</strong>
     </h3>
-
     <p>
-      The world of open source software depends on a lot of open source libraries. We are standing on the shoulders of giants, which helps us to reach further than we could otherwise.
+      <strong>Keeping a watch while you work.</strong>
     </p>
-
+    <!-- <%= render 'dashboard/monitoring_promo' %>-->
```
Comment (andrew) [inline app/views/pages/about.html.erb]: No need to keep this here

File: config/routes.rb [hunk 4]
```diff
@@ -178,6 +178,7 @@
 
   get '/about', to: 'pages#about', as: :about
   get '/privacy', to: 'pages#privacy', as: :privacy
+  get '/packagemanagercompatibility', to: 'pages#pmmatrix', as: :pmmatrix
```
Comment (andrew) [inline config/routes.rb]: What about `/compatibility` instead? Can we also change the controller action name  and template name

**Comment by BenJam** [comment]
hmm @andrew I might need a little assistance here.

File: app/views/pages/about.html.erb [hunk 5]
```diff
@@ -65,56 +78,51 @@
...
     </p>
     <%- end %>
-
+    <p>
+      Libraries.io tracks releases from <strong><%= number_with_delimiter(Project.total) %></strong> projects on <strong><%= PackageManager::Base.platforms.length %></strong> package managers in one place. Subscribe to a project and receive notifications of new versions or tags for platform like Bower and Go that don't store release information centrally. 
+    </p>
+    <h2>Supported Package Managers</h2>
     <hr>
+    <p>
+      Libraries.io supports the following package managers. For more detail check out <a href='<%= url_for(controller: 'pages', action: 'pmmatrix',) %>'>our compatibility matrix</a>.
```
Comment (andrew) [inline app/views/pages/about.html.erb]: name that you have a named route (`as: : pmmatrix`) you can use `pmmatrix_path` in the views, although that should be renamed to `compatibility_path` as well

**Comment by BenJam** [comment]
@andrew ready for review.

File: config/routes.rb (hunk 4 above)
Comment (BenJam) [inline config/routes.rb]: hahaa I knew you'd love le big slug 👍

File: app/views/pages/about.html.erb [hunk 6]
```diff
@@ -84,7 +83,7 @@
     <h2>Supported Package Managers</h2>
     <hr>
     <p>
-      Libraries.io supports the following package managers. For more detail check out <a href='<%= url_for(controller: 'pages', action: 'pmmatrix',) %>'>our compatibility matrix</a>.
+      Libraries.io supports the following package managers. For more detail check out <a href='<%= compatibility_path %>'>our compatibility matrix</a>.
```
Comment (andrew) [inline app/views/pages/about.html.erb]: Can you change `<a href='<%= compatibility_path %>'>our compatibility matrix</a>` to be a `link_to`?

Example: `<%= link_to 'our compatibility matrix', compatibility_path %>`

Comment (BenJam) [inline app/views/pages/about.html.erb]: ahhh i see http://api.rubyonrails.org/classes/ActionView/Helpers/UrlHelper.html

Comment (andrew) [inline app/views/pages/about.html.erb]: Using the named route helpers means that if we decide to move the action to a different controller, we won't need to go through the whole application and update every link pointing it.

**Pull request merged**